Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py

```diff
@@ -68,6 +69,63 @@ def test_get_incident_list(requests_mock):
     assert expected_output == outputs
 
 
+def test_get_starred_incident_list(requests_mock):
```

Comment:
move the tests to a class to encapsulate related tests of the same function/use case

Reply:
Done